extensions/resource-deployment/src/constants.ts

```diff
@@ -6,3 +6,13 @@
 export const DeploymentConfigurationKey: string = 'deployment';
 export const AzdataInstallLocationKey: string = 'azdataInstallLocation';
 export const ToolsInstallPath = 'AZDATA_NB_VAR_TOOLS_INSTALLATION_PATH';
+
+export enum ResourceTypeCategories {
+	ALL = 'All',
```

Comment:
No IoT?

Reply:
The PM has decided to not include it.

Reply:
Then make sure you remove them from the tags you have in the JSON